spec/rails_helper.rb

```diff
@@ -109,4 +115,24 @@ def delete_document; end
   c.default_cassette_options = { record: :new_episodes, re_record_interval: 7.days }
   c.ignore_hosts URI.parse(Rails.application.config_for(:elasticsearch).fetch("host")).host
   c.ignore_hosts 'pub.orcid.org', 'sandbox.orcid.org'
+
+  # FIXME: temporary workaround for feature specs
+  c.allow_http_connections_when_no_cassette = true
```

Comment:
What about testing other remote integrations, like e.g. with Zenodo? Wouldn't it impact those specs as well?